src/node_version.h

```diff
@@ -29,7 +29,7 @@
 #define NODE_VERSION_IS_LTS 0
 #define NODE_VERSION_LTS_CODENAME ""
 
-#define NODE_VERSION_IS_RELEASE 0
+#define NODE_VERSION_IS_RELEASE 1
```

Comment:
The version in this file needs to be set to v17.9.0

Reply:
Whoops, fixed!